internal/configs/module_call.go

```diff
@@ -202,6 +148,98 @@ func decodeModuleBlock(block *hcl.Block, override bool) (*ModuleCall, hcl.Diagno
 	return mc, diags
 }
 
+func (mc *ModuleCall) decodeStaticFields(eval *StaticEvaluator) hcl.Diagnostics {
+	if mc.Source == nil {
+		// This is an invalid module.  We already have error handling that has more context and can produce better errors in this
+		// scenario.  Follow the trail of mc.SourceAddr -> req.SourceAddr through the command package.
+		return nil
+	}
+
+	// Setup static variable mapping
```

Comment:
I think if we're leaving comments to split up the functionality of a method, we should just make 2 methods

- setupStaticVariableMapping
- decodeSourceField

I'm not a huge fan of these huge methods that do 2 things.